x-pack/dockerlogbeat/pipereader/reader_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipereader
+
+import (
+	"bytes"
+	"encoding/binary"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/docker/docker/api/types/plugins/logdriver"
+)
+
+func TestPipeReader(t *testing.T) {
```

Comment:
we are planning more tests? Also some fuzzing?

Reply:
yep!